client/verta/tests/unit_tests/test_deployed_model.py

```diff
@@ -379,3 +382,134 @@ def test_predict_400_error_message_missing(mocked_responses) -> None:
         '400 Client Error: Bad Request for url: '
         'https://test.dev.verta.ai/api/v1/predict/test_path at '
     )
+
+
+def test_batch_predict_with_one_batch_with_no_output_index(mocked_responses) -> None:
+    """ Call batch_predict with a single batch. """
+    expected_df = pd.DataFrame({"A": [1, 2, 3, 4, 5, 6, 7, 8, 9, 10], "B": [11, 12, 13, 14, 15, 16, 17, 18, 19, 20]})
+    expected_df_json = expected_df.to_dict(orient="index")
+    mocked_responses.post(
+        BATCH_PREDICTION_URL,
+        json=expected_df_json,
+        status=200,
+        headers={"verta-request-id": "hereISaTESTidFROMtheUSER"},
+        )
+    creds = EmailCredentials.load_from_os_env()
+    dm = DeployedModel(
+        prediction_url=PREDICTION_URL,
+        creds=creds,
+        token=TOKEN,
+        )
+    # the input below is entirely irrelevant since it"s smaller than the batch size
```

Comment:
did you cmd-F replace `'` with `"`

```suggestion
    # the input below is entirely irrelevant since it's smaller than the batch size
```

Reply:
ya